misc/theme-cicada/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.9
+PLUGIN_VERSION=		2.0
```

Comment:
you can do this if you want or use 1.10 if you want to reserve 2.0 for a larger update...